python/tests/kernel/test_cast_kernel.py

```diff
@@ -0,0 +1,170 @@
+# ============================================================================ #
```

Comment:
Looks like a couple old tests need a few tweaks.

Reply:
working on it!

Reply:
done!